carbon/assetlist.json

```diff
@@ -3,28 +3,228 @@
+        },
+        "coingecko_id": "carbon-usd"
+      },
+      {
+        "description": "BNB token on Carbon",
+        "denom_units": [
+          {
+            "denom": "bnb.1.6.773edb",
+            "exponent": 18,
+            "aliases": ["BNB"]
+          }
+        ],
+        "base": "bnb.1.6.773edb",
+        "name": "Binance Coin",
+        "display": "bnb",
+        "symbol": "BNB",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.svg"
+        },
+        "coingecko_id": "binancecoin"
+      },
+      {
+        "description": "bNEO token on Carbon",
+        "denom_units": [
+          {
+            "denom": "bneo.1.14.e2e5f6",
+            "exponent": 8,
+            "aliases": ["BNB"]
+          }
+        ],
+        "base": "bneo.1.14.e2e5f6",
+        "name": "BurgerNEO",
+        "display": "bneo",
+        "symbol": "bNEO",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/bneo.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/bneo.svg"
+        },
+        "coingecko_id": "neo"
+      },
+      {
+        "description": "BUSD (BEP-20) token on Carbon",
+        "denom_units": [
+          {
+            "denom": "busd.1.6.754a80",
+            "exponent": 18,
+            "aliases": ["BUSD"]
+          }
+        ],
+        "base": "busd.1.6.754a80",
+        "name": "BUSD (BEP-20)",
+        "display": "busd",
+        "symbol": "BUSD",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/busd.png"
+        },
+        "coingecko_id": "binance-usd"
+      },
+      {
+        "description": "ETH (ERC20) token on Carbon",
+        "denom_units": [
+          {
+            "denom": "eth.1.2.942d87",
+            "exponent": 18,
+            "aliases": ["ETH"]
+          }
+        ],
+        "base": "eth.1.2.942d87",
+        "name": "Ethereum (ERC20)",
+        "display": "eth",
+        "symbol": "ETH",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/eth.svg"
+        },
+        "coingecko_id": "ethereum"
+      },
+      {
+        "description": "stOSMO token on Carbon",
+        "denom_units": [
+          {
+            "denom": "ibc/75249A18DEFBEFE55F83B1C70CAD234DF164F174C6BC51682EE92C2C81C18C93",
+            "exponent": 6,
+            "aliases": ["stOSMO"]
+          }
+        ],
+        "base": "ibc/75249A18DEFBEFE55F83B1C70CAD234DF164F174C6BC51682EE92C2C81C18C93",
+        "name": "Stride Staked OSMO",
+        "display": "stOSMO",
+        "symbol": "stOSMO",
+        "logo_URIs": {
```

Comment:
For non-native assets, and especially ibc assets, could we please include 'traces' data? For something like ETH, how did it get to Carbon (synthetic vs bridge)? For something like stOSMO, how did it get to Carbon(just ibc transfer? what is the ibc hash input path(transfer/channel-##/denom)?)?